RevenueCatUI/CustomerCenter/Views/ManageSubscriptionsView.swift

```diff
@@ -0,0 +1,177 @@
+                SubscriptionDetailsView(subscriptionInformation: subscriptionInformation,
+                                        refundRequestStatus: viewModel.refundRequestStatus)
+            }
+
+            Spacer()
+
+            ManageSubscriptionsButtonsView(viewModel: viewModel,
+                                           openURL: openURL)
+        }
+        .task {
+            await checkAndLoadInformation()
+        }
+    }
+
+}
+
+@available(iOS 15.0, macOS 12.0, tvOS 15.0, watchOS 8.0, *)
+@available(macOS, unavailable)
+@available(tvOS, unavailable)
+@available(watchOS, unavailable)
+private extension ManageSubscriptionsView {
+
+    func checkAndLoadInformation() async {
+        if !viewModel.isLoaded {
+            await viewModel.loadSubscriptionInformation()
+            await viewModel.loadCustomerCenterConfig()
+        }
+    }
+
+}
+
+@available(iOS 15.0, macOS 12.0, tvOS 15.0, watchOS 8.0, *)
+@available(macOS, unavailable)
+@available(tvOS, unavailable)
+@available(watchOS, unavailable)
+struct HeaderView: View {
+    var body: some View {
+        Text("How can we help?")
+            .font(.title)
+            .padding()
+    }
+}
+
+@available(iOS 15.0, macOS 12.0, tvOS 15.0, watchOS 8.0, *)
+@available(macOS, unavailable)
+@available(tvOS, unavailable)
+@available(watchOS, unavailable)
+struct SubscriptionDetailsView: View {
+    let subscriptionInformation: SubscriptionInformation
+    let refundRequestStatus: String?
+
+    var body: some View {
+        VStack(alignment: .leading, spacing: 8) {
+            Text("\(subscriptionInformation.title) - \(subscriptionInformation.durationTitle)")
+                .font(.subheadline)
+                .padding([.horizontal, .top])
+
+            Text("\(subscriptionInformation.price)")
+                .font(.caption)
+                .foregroundColor(Color.gray)
+                .padding(.horizontal)
+
+            if let nextRenewal =  subscriptionInformation.nextRenewalString {
+                Text("\(subscriptionInformation.renewalString): \(String(describing: nextRenewal))")
+                    .font(.caption)
+                    .foregroundColor(Color.gray)
+                    .padding([.horizontal, .bottom])
+            }
+
+            if let refundRequestStatus = refundRequestStatus {
+                Text("Refund request status: \(refundRequestStatus)")
+                    .font(.caption)
+                    .bold()
+                    .foregroundColor(Color.gray)
+                    .padding([.horizontal, .bottom])
+            }
+        }
+    }
+}
+
+@available(iOS 15.0, macOS 12.0, tvOS 15.0, watchOS 8.0, *)
+@available(macOS, unavailable)
+@available(tvOS, unavailable)
+@available(watchOS, unavailable)
+struct ManageSubscriptionsButtonsView: View {
+
+    @ObservedObject
+    private(set) var viewModel: ManageSubscriptionsViewModel
+
+    let openURL: OpenURLAction
```

Comment:
Hmm this seems unused? (Maybe you're planning to use it later though)

Reply:
Oh yes, I had a contact support button, that I removed because it's not possible to configure it, and this is leftovers. Removing!